baseplate/server/__init__.py

```diff
@@ -20,6 +20,14 @@
 import traceback
 import warnings
 
+try:
+    # Python 3
+    from inspect import getfullargspec as getargspec
+except ImportError:
+    # Python 2
+    from inspect import getargspec
```

Comment:
`getfullargspec ` is pretty much a drop in replacement for `getargspec ` that was introduced in py 3. `getfullargspec` works with the following syntax while `getargspec` does not:
```
# the keyword argument `kwarg='bar'` used between `*args` and `**kwargs` is not valid in python 2.
def foo(app_config, arg, *args, kwarg='bar', **kwargs):
            pass
```

Reply:
can this go in the `_compat` module please?

Reply:
Rebased a removed py 2 support